client/lib/plans/features-list.js

```diff
@@ -547,15 +547,15 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_3GB_STORAGE ]: {
 		getSlug: () => constants.FEATURE_3GB_STORAGE,
-		getTitle: () => i18n.translate( '3GB Storage Space' ),
+		getTitle: () => i18n.translate( '3 GB Storage Space' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( '200 GB Storage Space' )` *ES Score: 7*